### DISCUSSION TITLE
ListBoxOptions are hidden

### DISCUSSION THREAD

**Discussion by mahouha** [tailwindlabs/headlessui#2191] [Help]
**What package within Headless UI are you using?**

 @headlessui/vue

**What version of that package are you using?**

v1.7.7

**What browser are you using?**

Chrome, Firefox

**Reproduction URL**

My project is local

**Describe your issue**

if I have multiple ListBoxOptions under eatch other, when I open a ListboxOptions, the options are not accessible and shown under the others

![Capture](https://user-images.githubusercontent.com/25373754/213113432-8f15ba9d-0ee2-4454-aa61-87c303c8beac.JPG)

**Comment by RobinMalfait** [comment]
Hey!

Please share a reproduction if you want to get some help on this. Headless UI itself is headless (comes with no styles) so that part is up to you. Depending on how you implemented things the solution will be different.

**Reply by mahouha** [reply]
Hello @RobinMalfait  !

I have implement a part of my app that contains Listboxs in StackBlitz if you can help me
[https://stackblitz.com/edit/vitejs-vite-2jbvke?file=src/App.vue](url)
